CMakeLists.txt

```diff
@@ -506,7 +508,7 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "Windows")
     else()
         # See https://gitlab.kitware.com/cmake/cmake/-/issues/18317 for following workaround
         STRING(REGEX REPLACE "/W3" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
-        ADD_COMPILE_OPTIONS(/nologo /WX /W4 /Gs32768 /D UNICODE /Gy /EHs-c- /GR- /GF /Z7 /Gw /Oy- /D_CRT_SECURE_NO_WARNINGS /wd4063)
+        ADD_COMPILE_OPTIONS(/nologo /WX /W4 /Gs32768 /D UNICODE /Gy /EHs-c- /GR- /GF /Z7 /Gw /Oy- /D_CRT_SECURE_NO_WARNINGS /wd4063 /wd4819)
```

Comment:
Why do we need to disable this warning?

Reply:
> Why do we need to disable this warning?

In my PC(Windows 10 Home), there are Unicode Warning in files library/os_stub/include/industry_standard/link_type_ex.h and unit_test/cmockalib/cmocka/cmocka.h.